frontend/lib/identification/identification_page.dart

```diff
@@ -45,14 +45,25 @@ class IdentificationPageState extends State<IdentificationPage> {
           return Container();
         }
 
+        final isStagingEnabled = Provider.of<SettingsModel>(context, listen: false).enableStaging;
+        final applicationUrl = getApplicationUrl(buildConfig.applicationUrl, isStagingEnabled);
         if (userCodeModel.userCodes.isNotEmpty) {
           final List<Widget> carouselCards = [];
           for (var code in userCodeModel.userCodes) {
+            final applicationUrlWithParameters = getApplicationUrlWithParameters(
```

Comment:
:thought_balloon: I think we only wanted to set parameters if the card is extendable

Reply:
i added this in the last commit and also adjusted the tests! thx for the hint
@seluianova 
Feel free to test